tests/test_models.py

```diff
@@ -724,7 +724,7 @@ def test_properties_access():
     elif VERSION[:2] > (2, 1):
```

Comment:
This elif seems to have become redundant now, but that can also be cleaned up later. The else branch can be deleted and the elif turned into an else, since we don't test anything below django 2.2.

I guess the issues with django main were fixed in another PR as well.

Reply:
Sounds good to drop it later, it seems to not be part of the PR changeset, so I think we can defer here.